.travis.yml

```diff
@@ -20,7 +20,8 @@ matrix:
     - rvm: jruby-1.7
     - rvm: jruby-9.1.17.0
     # See https://github.com/oracle/truffleruby/blob/master/doc/user/standalone-distribution.md#testing-truffleruby-in-travisci
-    - rvm: system
+    - name: truffleruby
```

Comment:
Oh, that's a cool trick, I'll update the docs :)

Reply:
Indeed! PR is already available at https://github.com/oracle/truffleruby/pull/1420 :smile: